nix/pyci.sh

```diff
@@ -0,0 +1,6 @@
+dune exec -- pip3 install .[dev]
+dune exec -- black .
+dune exec -- flake8 .
+# dune exec -- pytype .
+dune build test/py/unit/spec.py test/py/property/spec.py
+dune exec -- python3 _build/default/test/py/unit/spec.py
```

Comment:
Also you may want to put this inside a dune rule, actually Dune 3 supports directory targets so that would work pretty nicely I think? Depends on what the targets are.

Reply:
https://github.com/ejgallego/pycoq/blob/087d3e94e4195ab2fd01eba911bee13743bb8fe1/test/dune#L3 I think I already have a directory target? this seems to be working.